uaclient/config.py

```diff
@@ -249,12 +250,23 @@ def _attached_status(self) -> "Dict[str, Any]":
             response["expires"] = datetime.strptime(
                 contractInfo["effectiveTo"], "%Y-%m-%dT%H:%M:%SZ"
             )
+
+        resources = get_available_resources(self)
```

Comment:
The two places we call this we call it with `self` as the first argument from `UAConfig`; should it be a method on `UAConfig`?

(This doesn't need changing for us to land this PR, I'm just curious what you think.)

Reply:
There is a smell here, and I'm not sure how we should resolve it. get_available_resources is a utility function in the contract module and really relates to only contract client operations, so I don't feel like it belongs on the UAConfig class because we aren't really managing local system configuration. But, the fact that we are passing in the entire instance makes me feel like maybe we need to think instead about maybe passing in the outputs from get_available_resources as a parameter to UAConfig.status -> _attached_status and _unattached_status  to avoid this conflict/smell.